README.md

```diff
@@ -1303,6 +1310,7 @@ If you're looking for an entry point into domains like renewable energy, climate
 - [Terraso](https://github.com/techmatters/terraso-backend) - Develop an open source technology platform to help local community leaders practice Integrated Landscape Management.
 - [Open Soil Spectral Library](https://soilspectroscopy.github.io/ossl-manual/) - Brings together soil scientists, spectroscopists, informaticians, data scientists and software engineers to overcome some of the current bottlenecks preventing wider and more efficient use of soil spectroscopy.
 
+- [GainForest](https://github.com/GainForest) - A decentralised fund using artificial intelligence to measure and reward sustainable nature stewardship.
```

Comment:
Might not want to include at this time.

Reply:
Yes, as long as no open source project is released that fits to the guideline I would not include it.